docs/base/async_sdk.md

```diff
@@ -0,0 +1,217 @@
+---
+id: async_sdk
+title: Deta Base Python Async SDK
+---
+
+The Deta Base Async SDK can be used for reading and writing asynchronously with Deta Base.
+
+:::warning
+These are docs for an alpha version of the Python Deta Base Async SDK. The SDK API may change in a stable release.
+:::
+
+## Installing
+
+```
+pip install deta[async]==1.1.0a1
+```
+
+## Instantiating
+
+```py
+from deta import Deta
+
+# initialize with a project key
+# you can also init without specfying the project key explicitly
+# the sdk looks for the DETA_PROJECT_KEY env var in that case
+deta = Deta("project_key")
+
+# create an async base client
+async_db = deta.AsyncBase("base_name")
+```
+
+## Methods 
+
+The **`AsyncBase`** class offers the same API to interact with your Base as the **`Base`** class:
+
+### Put
+
+**`put(data: typing.Union[dict, list, str, int, float, bool], key:str=None):`**
+
+- **data** (required): The data to be stored
+- **key** (optional): The key to store the data under. It will be auto-generated if not provided.
+
+#### Example
+
+```py
+import asyncio
+from deta import Deta
+
+async_db = Deta("project_key").AsyncBase("base_name")
+
+async def put_item():
+	item = {"msg": "hello"}
+	await async_db.put(item, "test")
+	print("put item:", item)
+	await async_db.close()
+
+loop = asyncio.get_event_loop()
+loop.run_until_complete(put_item())
+```
+
+### Get
+
+**`get(key: str)`**
+
+- **key** (required): The key of the item to be retrieved
+
+#### Example
+
+```py
+import asyncio
+from deta import Deta
+
+async_db = Deta("project_key").AsyncBase("base_name")
+
+async def get_item():
+	item = await async_db.get("my_key")
+	print("got item:", item)
+
+	await async_db.close()
+
+loop = asyncio.get_event_loop()
+loop.run_until_complete(get_item())
+```
+
+### Delete
+
+**`delete(key: str)`**
+
+- **key** (required): The key of the item to delete 
+
+#### Example
+
+```py
+import asyncio
+from deta import Deta
+
+async_db = Deta("project_key").AsyncBase("base_name")
+
+async def del_item():
+	await async_db.delete("my-key")
+	print("Deleted item")
+	
+	await async_db.close()
+
+loop = asyncio.get_event_loop()
+loop.run_until_complete(del_item())
+```
+
+### Insert 
+
+`insert` is unique from `put` in that it will raise an error if the `key` already exists in the database whereas `put` overwrites the item.
```

Comment:
`insert` is unique from `put` in that it will raise an error if the `key` already exists in the database, whereas `put` overwrites the item.